golang/cmd/kafka-to-postgresql-v2/worker/analytics_test.go

```diff
@@ -235,3 +235,54 @@ func TestShiftDelete(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestStateAdd(t *testing.T) {
```

Comment:
in the future, you could create new tests using ginkgo directly, so then we would not need to convert them on  
a later stage. nothing to do here, just a note for future tests